### ISSUE TITLE
[BUG] Roblox not opening after launching from bloxstrap

### ISSUE THREAD

**Issue by OceanEmperor** [bloxstraplabs/bloxstrap#4093]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

2.8.2

### What problem did you encounter?

Whenever I open roblox with bloxstrap (to join a private server through browser) bloxstrap tries to launch roblox but when it's meant to open it just vanishes not opening. I tried reinstalling bloxstrap and roblox twice, both didnt work, checked error logs, which are fine too.

Bloxstrap just straight up doesnt open roblox, but from stock bootstrapper it works perfectly fine, probably an error within the code not letting the launcher open up roblox itself is my guess. 

### Bloxstrap Log

```text
N/A
```

**Comment by bluepilledgreat** [comment]
Does this happen with the stock Roblox bootstrapper?

**Comment by OceanEmperor** [comment]
@bluepilledgreat nope it works fine with it

**Comment by oskiebadoskie** [comment]
Having the same issue. Once bloxstrap says "Starting Roblox" it immediately crashes. I have not tweaked with fastflags or any major settings

**Comment by OceanEmperor** [comment]
yup same thing for me, havent changed a thing yet it just crashes

**Comment by truekvvrund** [comment]
ahh guys ig its a problem of new version just play for now at normal roblox then when bloxtrap will fix it then play with it bc i have the same problem

**Comment by OceanEmperor** [comment]
yea nah my shaders broke from this too, thanks bloxstrap (sarcasm)

**Comment by RandomRetartd** [comment]
I have nothing extra and Bloxstrap refuses to work, i click play on the site, doesn't work nor the Roblox app

**Comment by agusg12850** [comment]
osea que es un error y tenemos que esperar que lo solucionen?

**Comment by OceanEmperor** [comment]
@agusg12850 si

**Comment by OceanEmperor** [comment]
any updates?

**Comment by OceanEmperor** [comment]
okay so i updated to 2.8.3, it gave me this error:

System.UnauthorizedAccessException: Access to the path 'C:\Users\*user name*\AppData\Local\Bloxstrap\Versions\version-b71c150c7c1f40de\ucrtbase.dll' is denied.

@bluepilledgreat

**Comment by bluepilledgreat** [comment]
Assumed to be fixed.
